pkg/daemon/ceph/osd/volume.go

```diff
@@ -284,6 +285,10 @@ func (a *OsdAgent) initializeBlockPVC(context *clusterd.Context, devices *Device
 			// Add the cli argument for the metadata device
 			if metadataDev {
 				immediateExecuteArgs = append(immediateExecuteArgs, metadataArg...)
+				immediateExecuteArgs = append(immediateExecuteArgs, []string{
```

Comment:
could this be combined with the previous line so we only have a single `append()` instead of two inside the `if metadataDev` code block?